set.mm

```diff
@@ -425567,8 +425699,16 @@ that the number n of vertices in G is exactly k(k-1)+1.".  Variant of
     $( Value of operation ` F ` , mapping a vertex ` v ` and a positive integer
        ` n ` to the "(For a fixed vertex v, let f(n) be the number of) walks
        from v to v of length n" according to definition 5 in [Huneke] p. 2.
-       (Contributed by Alexander van der Vekens, 14-Sep-2018.)  (Revised by AV,
-       28-May-2021.) $)
+
+       ` ( X F N ) ` is not defined for ` N = 0 ` , since closed walks of
+       length 0 cannot be represented by a sequence (word) of vertices, see
+       ~ clwwlksn0 . ` ( X F N ) = (/) ` could have been explicitly defined,
+       but the operation value for operators for which the operation is not
+       defined is already ` (/) ` by definition (see ~ ndmovg ), therefore such
+       an extension of the definition of ` F ` is not necessary.  Nevertheless,
```

Comment:
Nevertheless --> In any case
?

Reply:
"In any case, the case ..." sounds a little bit clumsy.